src/Components/Calculator.js

```diff
@@ -1,24 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Calculator.css';
 import ClcScrn from './ClcScrn';
 import Btn from './Btn';
+import calculate from '../logic/calculate';
 
 const Calculator = () => {
+  const [result, setResult] = useState({});
+  const { total, next, operation } = result;
+
+  const onClick = (e) => {
+    const btn = e.target.innerHTML;
+    setResult(calculate(result, btn));
+  };
   const btnValues = [
     ['AC', '+/-', '%', '÷'],
     ['7', '8', '9', 'x'],
     ['4', '5', '6', '-'],
     ['1', '2', '3', '+'],
     ['0', '.', '='],
   ];
+
   return (
     <div className="clcBckgrnd">
-      <ClcScrn value="0" />
+      <ClcScrn value={total || next || operation || 0} />
       <div className="btnsPad">
```

Comment:
🚀  Fantastic job on implementing events on the calculator. 👏🏾 💯 

[OPTIONAL] Although everything works well on the calculator in the case of button events, I think it would be best if you improved the way the calculations are displayed on the calculator screen.

- Currently the calculator only displays the first digit. Normally the calculator should display everything including the operators and the next digits. It would be best to improve that in order to enhance user experience.

Reply:
Thank you so much.